src/page.js

```diff
@@ -0,0 +1,6 @@
+import {getQueriesForElement} from './get-queries-for-element'
+
+export const page =
+  typeof document !== 'undefined' && document.body
+    ? getQueriesForElement(document.body)
+    : null
```

Comment:
Would be nice to have an object that throws a descriptive error instead of a possible "cannot read property getByRole of null".

Something like 
```suggestion
    : Object.keys(queries).reduce((helpers, key) => {
        helpers[key] = () => throw new TypeError('For queries bound to document.body a global document has to be available... <Insert possible link to faq maybe?>')
        return helpers
      }, {})
```